vulnerabilities/force-feeding.md

```diff
@@ -0,0 +1,89 @@
+# Force Feeding
```

Comment:
Maybe a better title would be "Unexpected Ether Balance"? This way it focuses more on the vulnerability rather than the attack vector. Would have to update the below paragraph to reflect this

Reply:
What about "Unexpected Ether Transfer"?